addons/common/functions/fnc_getPosFromScreen.sqf

```diff
@@ -0,0 +1,40 @@
+#include "script_component.hpp"
+/*
+ * Author: mharis001
+ * Returns a world position based on the given screen position.
+ * Will attempt to find a flat position on an intersecting surface if possible.
+ *
+ * Arguments:
+ * 0: Screen Position <ARRAY> (default: getMousePosition)
```

Comment:
Does it really need to be wrapped into another array?

Reply:
Yes, cleaner `params`.